client/src/auth.ts

```diff
@@ -0,0 +1,26 @@
+import { createSlice } from "@reduxjs/toolkit";
+import { AppState } from "./store";
+
+export interface Auth {
+    isAuthenticated: boolean;
+}
+
+const initialState: Auth = {
+    isAuthenticated: true,
+};
+
+const authSlice = createSlice({
+    name: "auth",
+    initialState,
+    reducers: {
+        notAuthorized: (state) => {
+            state.isAuthenticated = false;
+        },
+    },
+});
+
+export const selectAuth: (state: AppState) => Auth = (state: AppState) => state.auth;
```

Comment:
this is a mutable selector, better to use createSelector

Reply:
created an issue